client/src/controllers/ActionController.ts

```diff
@@ -33,11 +33,11 @@ export class ActionController extends Controller {
     formElement.appendChild(csrftokenElement);
 
     if (this.redirectValue) {
-      const nextElement = document.createElement('input');
-      nextElement.type = 'hidden';
-      nextElement.name = 'next';
-      nextElement.value = window.location.href;
-      formElement.appendChild(nextElement);
+      const redirectElement = document.createElement('input');
+      redirectElement.type = 'hidden';
+      redirectElement.name = 'redirect';
+      redirectElement.value = window.location.href;
+      formElement.appendChild(redirectElement);
```

Comment:
I'm curious - why is this being renamed? Sorry if I missed the convo somewhere but I understood the convention with Django is always to call this a `next` param.

Reply:
Maybe we should make the value name `next`?

Reply:
Yes
In the previous implementation, the argument was named `next` in the LockUnlock function. Thibaud and I discussed changing that to something else that is easier to understand what it means. But changing the dynamically created input element name to next messes with the redirect url. It always takes me to page 60. But using next as the name, ebverything works. So I think I'll just leave that part and not change it

Reply:
Ok. Yeah maybe it's best to keep it as `next` for both the value name and the actual form field name.